tools/run-tests.py

```diff
@@ -108,7 +108,8 @@ def skip_if(condition, desc):
 
 # Test options for test262-esnext
 TEST262_ESNEXT_TEST_SUITE_OPTIONS = [
-    Options('test262_tests_esnext', OPTIONS_PROFILE_ESNEXT + ['--line-info=on', '--error-messages=on']),
+    Options('test262_tests_esnext', OPTIONS_PROFILE_ESNEXT
+            + ['--line-info=on', '--error-messages=on', '--cpointer-32bit=on', '--mem-heap=65536']),
```

Comment:
Do we need the `cpointer-32bit` at all?

Reply:
No, 20M heap is enough without cpointer32.